templates/main.template

```diff
@@ -100,165 +126,239 @@ Parameters:
-      end with a hyphen (-).
+      end with a hyphen (-). If you are unsure, do not change this value.
     Default: aws-quickstart
     Description: S3 bucket name for the Quick Start assets. Quick Start bucket name
       can include numbers, lowercase letters, uppercase letters, and hyphens (-).
-      It cannot start or end with a hyphen (-).
+      It cannot start or end with a hyphen (-). If you are unsure, do not change this value.
     Type: String
   QSS3KeyPrefix:
     AllowedPattern: ^[0-9a-zA-Z-/]*$
     ConstraintDescription: Quick Start key prefix can include numbers, lowercase letters,
-      uppercase letters, hyphens (-), and forward slash (/).
+      uppercase letters, hyphens (-), and forward slash (/). If you are unsure, do not change this value.
     Default: quickstart-compliance-pci/
     Description: S3 key prefix for the Quick Start assets. Quick Start key prefix
       can include numbers, lowercase letters, uppercase letters, hyphens (-), and
-      forward slash (/).
+      forward slash (/). If you are unsure, do not change this value.
     Type: String
 Mappings:
-  CustomVariables:
-    vResourceEnvironmentTagKey:
-      Value: Environment
-    vResourceEnvironmentTagValue:
-      Value: development
   RegionServiceSupport:
     ap-northeast-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ap-northeast-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ap-south-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ap-southeast-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ap-southeast-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     ca-central-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     eu-central-1:
       NatGateway: 'true'
-      ConfigRules: 'true'
       Glacier: 'true'
     eu-west-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     eu-west-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     sa-east-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'false'
     us-east-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     us-east-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     us-gov-west-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     us-west-1:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
     us-west-2:
-      ConfigRules: 'true'
       NatGateway: 'true'
       Glacier: 'true'
+#these have been updated to a 2019 version
   AWSAMIRegionMap:
     AMI:
-      AMZNLINUXHVM: amzn-ami-hvm-2018.03.0.20181129-x86_64-gp2
+      AMZNLINUXHVM: amzn-ami-hvm-2018.03.0.20190514-x86_64-gp2
     ap-northeast-1:
-      AMZNLINUXHVM: ami-00a5245b4816c38e6
+      AMZNLINUXHVM: ami-0ccdbc8c1cb7957be
```

Comment:
Was this a manual update? We'll go through and update these automatically as well. :)

Reply:
lol ah man, yes this was all manual. Good to know.